x-pack/platform/plugins/shared/spaces/test/scout/api/tests/spaces_only/import_in_single_space.spec.ts

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RoleApiCredentials } from '@kbn/scout';
+import { expect, tags } from '@kbn/scout';
+
+import {
+  ATTRIBUTE_TITLE_KEY,
+  ATTRIBUTE_TITLE_VALUE,
+  COMMON_HEADERS,
+  DASHBOARD_SAVED_OBJECT,
+  SPACES,
+  TEST_SPACES,
+} from '../../constants';
+import { apiTest } from '../../fixtures';
+import { prepareImportFormData } from '../../helpers';
+
+/**
+ * Tests for the Saved Objects Import API (_import) within individual spaces (default space and a "space_1" space).
+ *
+ * This test suite validates import behavior when operating within a single space at a time to ensure
+ * consistent behavior regardless of the space context. The test suite uses an API key with minimal 'savedObjectsManagement' privileges to ensure
+ * proper role-based access control.
+ *
+ * Note: since version 8.0, Kibana requires most saved objects to have globally unique IDs
+ * Learn more: https://www.elastic.co/docs/explore-analyze/find-and-organize/saved-objects
+ */
+TEST_SPACES.forEach((space) => {
+  const spacePath = space.spaceId === 'default' ? '' : `s/${space.spaceId}/`;
+
+  apiTest.describe(`_import API within the ${space.name} space`, { tag: tags.ESS_ONLY }, () => {
+    let savedObjectsManagementCredentials: RoleApiCredentials;
+
+    apiTest.beforeAll(async ({ kbnClient, log, requestAuth }) => {
+      // Create the space (skip for default which always exists)
+      if (space.spaceId !== SPACES.DEFAULT.spaceId) {
+        log.info(`Creating space [${space.spaceId}] for test suite`);
+        await kbnClient.spaces.create({
+          id: space.spaceId,
+          name: space.name,
+          description: space.description,
+          disabledFeatures: [...space.disabledFeatures],
+        });
+      } else {
+        log.info(`Using default space for test suite`);
```

Comment:
Same question, I would move it to `log.debug(...)`

Reply:
I have replaced all `info` logs with `debug` in [613e970](https://github.com/elastic/kibana/pull/244323/commits/613e97057cbd564e5392df7d7cd3b351704c1378).